warehouse/query-core/src/main/java/datawave/query/config/Profile.java

```diff
@@ -12,31 +12,31 @@
 /**
  * Purpose: Provides a mechanism to provide a user configurable way to tune a user's query, such that he or she may take advantage of features that could have
  * negative implications on other user queries.
- * 
+ *
  * Design: Simple interface. User would select the profile through a bean
- * 
+ *
  *
```

Comment:
Small thing, could you add the <T> generic in the annotation with a @param tag? I think maven picks those up with warnings if they aren't included

Reply:
I made these changes locally while I await feedback. Shamefully/Honestly this isn't something I ever really thought about while writing javadoc, so I really appreciate you pointing it out and helping me clue into it.